src/tests/e2e.ts

```diff
@@ -1,141 +1,130 @@
-    // for breakpoints
-    await browser.driver.wait(async () => {
-      return await debugToolbar.isDisplayed();
-    }, 2 * defaultTimeout, "debug toolbar not found -- timed out");
-
-    sendTrafficToPod(debugToolbar);
-    debugToolbar.waitForBreakPoint();
-  });
+
+/**
+ * This suite tests basic flow of mirroring traffic from remote pod.
+ * - Enable mirrord
+ * - Start debugging the python file
+ * - Select the pod from the QuickPick
+ * - Send traffic to the pod
+ * - Tests successfully exit if "GET: Request completed" is found in the terminal
+*/
+describe("mirrord sample flow test", function () {
+
+    this.timeout("6 minutes"); // --> mocha tests timeout
+    this.bail(true); // --> stop tests on first failure
+
+    let browser: VSBrowser;
+
+    const testWorkspace = join(__dirname, '../../test-workspace');
+    const fileName = "app_flask.py";
+    const defaultTimeout = 10000; // = 10 seconds
+
+    before(async function () {
+        console.log("podToSelect: " + podToSelect);
+        console.log("kubeService: " + kubeService);
+
+        expect(podToSelect).to.not.be.undefined;
+        expect(kubeService).to.not.be.undefined;
+
+        browser = VSBrowser.instance;
+        // need to bring the flask app in open editors
+        await browser.openResources(testWorkspace, join(testWorkspace, fileName));
+    });
+
+    it("enable mirrord", async function () {
+        const statusBar = new StatusBar();
+
+        await browser.driver.wait(async () => {
+            return await statusBar.isDisplayed();
+        });
+
+        await browser.driver.wait(async () => {
+            for (let button of await statusBar.getItems()) {            
+                if ((await button.getText()).startsWith('mirrord')) {                
+                    await button.click();
+                    return true;
+                }
+            }
+        }, defaultTimeout, "mirrord `enable` button not found -- timed out");
+    });
+
+    it("select pod from quickpick", async function () {
+        await startDebugging();
+        const inputBox = await InputBox.create(defaultTimeout * 2);
+        // assertion that podToSelect is not undefined is done in "before" block   
+        await browser.driver.wait(async () => {
+            if (!await inputBox.isDisplayed()) {
+                return false;
+            }
+
+            for (const pick of await inputBox.getQuickPicks()) {
+                let label = await pick.getLabel();
+
+                if (label === podToSelect) {
+                    return true;
+                }
+
+                if (label === "Show Pods") {
+                    await pick.select();
+                }
+            }
+
+            return false;
+        }, defaultTimeout * 2, "quickPick not found -- timed out");
+
+        await inputBox.selectQuickPick(podToSelect!);
+    });
+
+    it("wait for process to write to terminal", async function () {
```

Comment:
```suggestion
    it("wait for process to write to terminal", async () => {
```

Or change the others to use `function`